subcommands/targets/show.go

```diff
@@ -71,12 +85,7 @@ func doShow(cmd *cobra.Command, args []string) {
 		}
 		fmt.Println()
 		// Tabby can't do indented tables, so...
-		longestNameLen := 0
-		for name := range target.ComposeApps {
-			if len(name) > longestNameLen {
-				longestNameLen = len(name)
-			}
-		}
+		longestNameLen, sortedApps := sortedAppsNames(target)
```

Comment:
Ah funny, you sorted the compose-apps.
I was actually going to mention this next, but i originally meant the call to:
```
hashes, targets := getTargets(factory, version)
```
on line 51 of the original file probably needed to be sorted as the Target Names come up in random orders as well.

Reply:
here you go - https://github.com/foundriesio/fioctl/pull/109/commits/079187824d044e3df2bdd551986324e422ed1a18